tool/gravity/cli/commands.go

```diff
@@ -1528,6 +1541,16 @@ type SystemStreamRuntimeJournalCmd struct {
 	*kingpin.CmdClause
 }
 
+// SystemSelinuxBootstrapCmd configures SELinux file contexts and ports on the node
+type SystemSelinuxBootstrapCmd struct {
```

Comment:
Since the policy is automatically bootstrapped - there's no way to do this manually with ease. This will allow to generate the local filecontext/port changes script to run along with the policy module.
I might be adding additional command to extract the policy modules as I cannot completely exclude the possibility of something breaking and us needing to load policy manually.